docs/manipulation.rst

```diff
@@ -315,6 +315,31 @@ and True elsewhere. It is this way to be consistent with ``astropy.nddata``.
 .. note:: The mask attribute is the only attribute modified by ``snr_threshold()``. To
              retrieve the masked flux data use ``spectrum.masked.flux_masked``.
 
+Shifting
+--------
+
+To shift a spectrum in wavelength, one can use a construct like this:
+
+.. code-block:: python
+
+    >>> from specutils import Spectrum1D
+    >>> wavelengths = np.arange(0, 10) * u.um
+    >>> flux = 100 * np.abs(np.random.randn(10)) * u.Jy
+    >>> spectrum = Spectrum1D(spectral_axis=wavelengths, flux=flux)
+    >>> spectrum  #doctest:+SKIP
```

Comment:
why is this ``#doctest:+SKIP``? (same question for the ``new_spec`` line below)  I'd suggest instead maybe instead using ``FLOAT_CMP`` or ``ELLIPSES`` if the problem is floating-point precision.

Reply:
That's right. ELLIPSIS works better there.